.github/workflows/ci.yml

```diff
@@ -350,3 +350,46 @@ jobs:
           CIBW_BUILD_VERBOSITY: 1
         with:
           package-dir: examples/namespace_package
+
+  test-mingw:
+    runs-on: windows-latest
+    name:  ${{ matrix.python-version }} mingw-${{ matrix.arch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [
+          { msystem: MINGW64, arch: x86_64, path: mingw64, rust_target: x86_64-pc-windows-gnu },
+          { msystem: MINGW32, arch: i686, path: mingw32, rust_target: i686-pc-windows-gnu }
+        ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install MSys2 and dependencies
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          msystem: ${{ matrix.msystem }}
+          install: >-
+            git
+            mingw-w64-${{ matrix.arch }}-python
+            mingw-w64-${{ matrix.arch }}-python-pip
+            mingw-w64-${{ matrix.arch }}-openssl
+            mingw-w64-${{ matrix.arch }}-toolchain
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: ${{ matrix.rust-target }}
```

Comment:
I think we should install Rust in msys2 shell or build it with `--target x86_64-pc-windows-gnu`.

Looks like it's using `x86_64-pc-windows-msvc` because it's the default toolchain.